klog.go

```diff
@@ -1077,7 +1077,17 @@ func (sb *syncBuffer) rotateFile(now time.Time, startup bool) error {
 	}
 	var err error
 	sb.file, _, err = create(severityName[sb.sev], now, startup)
-	sb.nbytes = 0
+	if startup {
+		fileInfo, err := sb.file.Stat()
+		if err != nil {
+			return err
```

Comment:
does this need a slightly more descriptive error message - e.g. `fmt.Errorf("..... could not ....: %v, err)`?